### PR TITLE
Rework tokens

### PR DESCRIPTION
**Pull request by stiaan**
Changes proposed in this pull request:

* 
* 
*

### CODE REVIEW

File: design-tokens/force-base/border-color.yml [hunk 1]
```diff
@@ -11,28 +11,28 @@ imports:
...
     cssProperties:
     - 'border*'
     - box-shadow
     - outline
     - outline-color
     comment: Our product brand blue.
   COLOR_BORDER_BRAND_DARK:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     comment: 'Our product brand blue, darkened to meet accessibility color contrast ratios with white text.'
```
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: @stiaan a very small detail, but for consistency sake i think we should edit this comment for `COLOR_BORDER_BRAND_DARK` too to say that our product brand is green.

File: design-tokens/force-base/border-color.yml [hunk 2]
```diff
@@ -150,17 +150,17 @@ props:
     value: '{!WHITE}'
     comment: These borders create the faux checkmark when the checkbox toggle is in the checked state.
   COLOR_STROKE_BRAND:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     cssProperties:
     - stroke
     comment: Our product brand blue.
```
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: @stiaan same here; change comment to reflect the new color value

File: design-tokens/force-base/border-color.yml [hunk 3]
```diff
@@ -150,17 +150,17 @@ props:
...
   COLOR_STROKE_BRAND_HOVER:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     cssProperties:
     - stroke
     comment: Hover stroke color for our product brand blue.
```
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: @stiaan another required comment change

File: design-tokens/force-base/border-color.yml [hunk 4]
```diff
@@ -150,17 +150,17 @@ props:
...
   COLOR_STROKE_BRAND_ACTIVE:
-    value: '{!BISCAY}'
+    value: '{!PICSA_CHARCOAL}'
     cssProperties:
     - stroke
     comment: Active stroke color for our product brand blue.
```
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: @stiaan `blue` to `charcoal`..?

File: package.json [hunk 5]
```diff
@@ -78,6 +78,7 @@
     "gulp-sass": "3.1.0",
     "gulp-sourcemaps": "2.4.1",
     "gulp-stylelint": "3.9.0",
+    "gulp-theo": "^1.0.0-alpha.4",
```
Comment (NicolaasDieBaas) [inline package.json]: @stiaan is which way are we using Theo currently..?

**Comment by NicolaasDieBaas** [comment]
@stiaan I added some minor comments to specific lines of code containing changes you made.  Please inform me if you can't see them.  
I've noticed that you have changed quite a couple of `box-shadow` properties to `none`.  Why would this be?  Just curious to know.
When you've made the required small text changes, I'm happy for @imsickofmaps to have a quick glance-over too before we go ahead and land.

File: design-tokens/force-base/border-color.yml (hunk 1 above)
Comment (stiaan) [inline design-tokens/force-base/border-color.yml]: @NicolaasDieBaas  Cool. I will look into proper use and organisation of colours and comments with the next design tasks.

File: design-tokens/force-base/border-color.yml (hunk 4 above)
Comment (stiaan) [inline design-tokens/force-base/border-color.yml]: @NicolaasDieBaas ? As part of my exploration into how SLDS is put together, I made changes to colors and things with no intent of keeping it that way. In this case, I replaced all instances of Biscay with charcoal. We shouldn't worry too much about these for now. But I will look into proper organisation of color schemes as part of another task.

File: package.json (hunk 5 above)
Comment (stiaan) [inline package.json]: @NicolaasDieBaas  It is what is recommended on the SLDS github repo: https://github.com/salesforce-ux/theo

File: design-tokens/force-base/border-color.yml (hunk 1 above)
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: great thanks

File: design-tokens/force-base/border-color.yml (hunk 4 above)
Comment (NicolaasDieBaas) [inline design-tokens/force-base/border-color.yml]: @stiaan ok great

File: package.json (hunk 5 above)
Comment (NicolaasDieBaas) [inline package.json]: @stiaan ok so if I understand correctly, we're not using it currently but will very probably do so in the future

File: design-tokens/force-base/border-color.yml (hunk 1 above)
Comment (stiaan) [inline design-tokens/force-base/border-color.yml]: @NicolaasDieBaas I've made this change. Will commit shortly with some of the others.

**Comment by stiaan** [comment]
@NicolaasDieBaas 

- I've made a commit with those comments added
- Re: the box-shadow. Whilst in process of modifying design-tokens, the box shadows often produced an undesired result (e.g. a green glow around an error box input etc) so I've set them to "none" for now to speed things up. However, I have a strong suspicion that we will remove them completely in the future as we move towards a more clean style for inputs.

File: package.json (hunk 5 above)
Comment (stiaan) [inline package.json]: @NicolaasDieBaas Not sure. @imsickofmaps was this part of what you added to run the styles to write to index.css?

Comment (imsickofmaps) [inline package.json]: @stiaan I don't think I added anything to the `package.json` just to the script. I can't see anything we use dependant on it on npm: https://www.npmjs.com/package/gulp-theo 

Last thing to check would be uninstall it and see if you can still build?

Comment (stiaan) [inline package.json]: @imsickofmaps @NicolaasDieBaas  I've uninstalled it and the build happens. However, it doesn't show me that a file has been changed to remove it (package.json) but we can check again next time.

Comment (NicolaasDieBaas) [inline package.json]: @stiaan did you do a `npm uninstall gulp-theo` AND manually deleted the relevant line from `package.json`..?

**Comment by stiaan** [comment]
@NicolaasDieBaas I've resolved the conflict. Can I merge this in?

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**